velox/exec/PrefixSort.cpp

```diff
@@ -39,8 +39,15 @@ FOLLY_ALWAYS_INLINE void encodeRowColumn(
   } else {
     value = *(reinterpret_cast<T*>(row + rowColumn.offset()));
   }
-  prefixSortLayout.encoders[index].encode(
-      value, prefixBuffer + prefixSortLayout.prefixOffsets[index]);
+  if constexpr (std::is_same_v<T, StringView>) {
+    prefixSortLayout.encoders[index].encode(
```

Comment:
You could encode null varchar here and add a function encodeNoNulls(const StringView& value) to avoid copy StringView.

Reply:
Or you could add a function encodeNullStringView in PrefixSortEncoder.h